pkgs/by-name/gr/graphene-hardened-malloc/update.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env nix-shell
+#!nix-shell --pure -i bash -p cacert curl common-updater-scripts
```

Comment:
```suggestion
#!nix-shell --pure -i bash -p bash cacert curl common-updater-scripts
```
I forgot bash itself will needed to be added here since it's pure now